src/components/pipelineRuns-details/PipelineRunDetailsPage.tsx

```diff
@@ -0,0 +1,252 @@
+            </div>
+          </Tooltip>
+        )}
+        {(pipelineRun?.metadata?.annotations?.[
+          DELETED_RESOURCE_IN_K8S_ANNOTATION
+        ] === 'true' ||
+          pipelineRun?.metadata?.annotations?.[
+            RESOURCE_LOADED_FROM_RESULTS_ANNOTATION
+          ] === 'true') && (
+          <Tooltip content={t('Archived in Tekton results')}>
+            <ArchiveIcon className="pipelinerun-details-page__results-indicator" />
+          </Tooltip>
+        )}
+        <ResourceStatus>
+          <Status
+            status={pipelineRunFilterReducer(pipelineRun)}
+            title={pipelineRunTitleFilterReducer(pipelineRun)}
+          />
+        </ResourceStatus>
+      </div>
+    );
+  }, [pipelineRun]);
+  if (!pipelineRunLoaded) {
+    return <LoadingBox />;
+  }
+  return (
+    <DetailsPage
+      obj={pipelineRun}
+      headTitle={name}
+      title={<Text component={TextVariants.h1}>{resourceTitleFunc}</Text>}
+      model={PipelineRunModel}
+      breadcrumbs={[
+        <BreadcrumbItem key="app-link" component="div">
+          <Link
+            data-test="breadcrumb-link"
+            className="pf-v5-c-breadcrumb__link"
+            to={`/pipelines/ns/${namespace}/pipeline-runs`}
+          >
+            {t('PipelineRuns')}
+          </Link>
+        </BreadcrumbItem>,
+        {
+          path: `/pipelines/ns/${namespace}/`,
+          name: t('PipelineRun details'),
+        },
+      ]}
+      pages={[
+        navFactory.details(PipelineRunDetails),
+        navFactory.editYaml(ResourceYAMLEditorViewOnly),
+        {
+          href: 'parameters',
+          name: t('Parameters'),
+          component: (pageProps) => (
+            <PipelineRunParametersForm obj={pipelineRun} {...pageProps} />
+          ),
+        },
+        {
+          href: 'logs',
+          name: t('Logs'),
+          component: PipelineRunLogsWithActiveTask,
+        },
+        navFactory.events(PipelineRunEvents),
+      ]}
+      actions={[
+        {
+          key: 'rerun-pipelineRun',
+          label: t('Rerun'),
+          onClick: () => reRunAction(),
+        },
+        {
+          key: 'stop-pipelineRun',
+          label: t('Stop'),
+          tooltipProps: {
+            content: t(
+              'Let the running tasks complete, then execute finally tasks',
+            ),
+          },
+          onClick: () => stopAction(),
+          hidden: shouldHidePipelineRunStop(pipelineRun, PLRTasks),
+        },
+        {
+          key: 'cancel-pipelineRun',
+          label: 'Cancel',
+          tooltipProps: {
+            content: t(
+              'Interrupt any executing non finally tasks, then execute finally tasks',
+            ),
+          },
+          onClick: () => cancelAction(),
+          hidden: shouldHidePipelineRunCancel(pipelineRun, PLRTasks),
+        },
+        {
+          key: 'delete-pipelineRun',
+          label: t('Delete PipelineRun'),
+          onClick: () => launchDeleteModal(),
```

Comment:
We have to disable it if it is deleted in k8s along with a tooltip "Resource is being fetched from Tekton Results."